src/lib/metrics.ts

```diff
@@ -408,7 +419,6 @@ export default class MetricsMonitor {
                     instanceOnboardingMetrics,
                     projectsOnboardingMetrics,
                 ] = await Promise.all([
```

Comment:
We do not need to run these in parallel. I would be to get rid of promise all.

Reply:
Exactly, we are going to do that gradually. When we register a new metric into this new implementation, the new implementation will run them sequentially